### PR TITLE
Update kubevirt sriov lane

### PR DESCRIPTION
**Pull request by qinqon**
Signed-off-by: Quique Llorente <ellorent@redhat.com>

### CODE REVIEW

File: github/ci/prow/files/jobs/kubevirt/kubevirt-presubmits.yaml [hunk 1]
```diff
@@ -455,7 +455,14 @@ presubmits:
         - "/usr/local/bin/runner.sh"
         - "/bin/sh"
         - "-c"
-        - "apt update && apt install gettext-base -y && export TARGET=kind-k8s-sriov-1.14.2 && automation/test.sh"
+        - >
+            apt update &&
+            apt install gettext-base -y &&
+            wget https://dl.google.com/go/go1.13.8.linux-amd64.tar.gz &&
```
Comment (phoracek) [inline github/ci/prow/files/jobs/kubevirt/kubevirt-presubmits.yaml]: Would it make sense to install Go in the base image `gcr.io/k8s-testimages/bootstrap:v20190516-c6832d9` instead? Ram did it in his https://github.com/kubevirt/project-infra/pull/373

Comment (qinqon) [reply github/ci/prow/files/jobs/kubevirt/kubevirt-presubmits.yaml]: Didn't want to change image, can we use the kubevirt bootstram image here ?

Comment (phoracek) [reply github/ci/prow/files/jobs/kubevirt/kubevirt-presubmits.yaml]: There is a different image for kubevirt with Go installed? I'd go for that.

Comment (rmohr) [reply github/ci/prow/files/jobs/kubevirt/kubevirt-presubmits.yaml]: Why do we need gettext and all this?

**Comment by qinqon** [comment]
The new operator uses envsusbst and algo has some go tools to do the certificate fixing on webhooks.

**Comment by qinqon** [comment]
Already done at master

**Pull request closed without merging**